web_notify/static/src/js/services/notification_services.esm.js

```diff
@@ -0,0 +1,53 @@
+import {markup} from "@odoo/owl";
+import {registry} from "@web/core/registry";
+
+export const webNotificationService = {
+    dependencies: ["bus_service", "notification", "action"],
+
+    start(env, {bus_service, notification: notificationService, action}) {
+        function displaywebNotification(notification) {
```

Comment:
```suggestion
        function displayWebNotification(notification) {
```
on this occasion: renamed displaywebNotification to displayWebNotification for camelCase consistency in Javascript.